go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/19chonm/461_1_23
+
+go 1.19
```

Comment:
Do you know if we have go 1.19 available on eceprog? If I run `go version` I get `go version go1.18.4 linux/amd64`.